TeamCreator/Modules/Entry/View/EntryViewController.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  EntryViewController.swift
+//  TeamCreator
+//
+//  Created by Yunus Emre ÖZŞAHİN on 29.07.2024.
+//
+import UIKit
+
+protocol EntryViewControllerProtocol: AnyObject {
+    func reloadCollectionView()
+    func navigateToSecond()
+}
+
+final class EntryViewController: BaseViewController {
+
+    //MARK: - Outlets
```

Comment:
```suggestion
    // MARK: - Outlets
```
diğer markları da bu şekilde düzeltebilir miyiz